tasks/cluster_agent.py

```diff
@@ -0,0 +1,133 @@
+"""
+Cluster Agent tasks
+"""
+
+import os
+import glob
+import shutil
+
+from invoke import task
+import invoke
+from invoke.exceptions import Exit
+
+from .build_tags import get_build_tags
+from .utils import get_build_flags, bin_name
+from .utils import REPO_PATH
+from .go import deps
+
+#constants
+BIN_PATH = os.path.join(".", "bin", "cluster-agent")
+AGENT_TAG = "datadog/cluster_agent:master"
+
+@task
+def build(ctx, rebuild=False, race=False, static=False, use_embedded_libs=False):
+    """
+    Build Cluster Agent
+
+     Example invokation:
+        inv cluster-agent.build
+    """
+
+    build_tags = get_build_tags("all", ["snmp", "cpython"])
```

Comment:
You should use the same logic as dsd: have a list of whitelisted tags you want to enable, instead of blacklisting what you don't want.
That way, when we add new tags, they won't automatically be added in that product. This list should be pretty short for now.

When you rebase on master, you should take `kubeapiserver`